stylesheets/components/card.css

```diff
@@ -0,0 +1,122 @@
+.card {
+    font-family: 'Poppins', sans-serif;
+    margin: var(--margin-one);
+    display: flex;
+    max-width: 19rem;
+    height: fit-content;
+    background-color: var(--white);
+    flex-direction: column;
+    border-radius: 0.5rem;
+    box-shadow: 0.25rem 0.25rem 1rem var(--gray-400);
+}
+
+.card-img {
+    cursor: pointer;
+    border-radius: 0.5rem;
+    max-height: 10rem;
+}
+
+.card-title {
+    margin: 0.5rem var(--margin-one);  
+}
+
+.card-info {
+    margin: 0 var(--margin-one);
+}
+
+.card-title p {
+    margin-top: 0.5rem;
+    font-weight: bold;
+}
+
+.card-icon {
+    width: 2rem;
+    padding: 0.25rem 1.75rem 0.25rem 0.5rem;
```

Comment:
Again, had it been in `px` it would be much more readable

Reply:
Updated in the latest commit